extensions/amp-video-iframe/validator-amp-video-iframe.protoascii

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-video-iframe
+  html_format: AMP  # Iframes not allowed in ads.
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-video-iframe"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+attr_lists: {
+  name: "amp-video-iframe-common"
+  attrs: { name: "album" }
+  attrs: { name: "alt" }
+  attrs: { name: "artist" }
+  attrs: { name: "artwork" }
+  attrs: { name: "attribution" }
+  attrs: {
+    name: "autoplay"
+    value: ""
+  }
+  attrs: {
+    name: "implements-media-session"
+    value: ""
+  }
+  attrs: {
+    name: "implements-rotate-to-fullscreen"
+    value: ""
+  }
+  attrs: {
+    name: "preload"
+    value: "auto"
+    value: "metadata"
+    value: "none"
+    value: ""
+  }
+  attrs: {
+    name: "rotate-to-fullscreen"
+    value: ""
+  }
+  attrs: {
```

Comment:
wanna do `dock` for this now or later? (I think `dock` should launch with amp-video, amp-video-iframe and amp-youtube support as MVP)